config/software/omnibus-toolchain.rb

```diff
@@ -20,14 +20,18 @@
 license :project_license
 
 # gnu utilities
-dependency "patch"
-dependency "bash"
-dependency "make"
-dependency "cacerts"
-dependency "gtar"
+if windows?
+  dependency "msys2core_x64"
```

Comment:
Don't we need a `msys2core_x86` software def also that is included if we are in 32-bit mode? The `windows_arch_i386?` helper is available for [use in software definitions](https://github.com/chef/omnibus-software/search?utf8=%E2%9C%93&q=windows_arch_i386%3F).